### PR TITLE
Troubleshooting guide update

### PR DESCRIPTION
**Pull request by xu-yuan1**
### Details:
 - *update based on comments from https://github.com/openvinotoolkit/openvino/pull/11785*

### Tickets:
 - *CVS-83057*

### CODE REVIEW

File: docs/install_guides/troubleshooting-steps.md [hunk 1]
```diff
@@ -4,23 +4,23 @@ If you run into issues while installing or configuring OpenVINO™, you can try
 
 ## Check the versions of OpenVINO Runtime and Developement Tools
 
-* To check the version of OpenVINO Development Tools, use the following command:
+* To check the version of OpenVINO Development Tools, use the following command: <!--updated-->
    ```sh
    mo --version
    ```
-* To check the version of OpenVINO Runtime, use the following command:
+* To check the version of OpenVINO Runtime, use the following command: <!--updated-->
```
Comment (helena-intel) [inline docs/install_guides/troubleshooting-steps.md:11]:
```suggestion
* To check the version of OpenVINO Runtime, use the following code: <!--updated-->
```

File: docs/install_guides/troubleshooting-steps.md [hunk 2]
```diff
@@ -44,23 +44,29 @@ For Ubuntu and RHEL 8 systems, if you installed OpenVINO Runtime via the install
...
+<!--updated-->
+- If you have installed OpenVINO before, you probably have added `setupvars` to your `PATH /.bashrc` or Windows environment variables. After restarting your environment, you should see similar information as below: 
 ```sh
 [setupvars.sh] OpenVINO™ environment initialized
 ```
    - If you don't see the information, your PATH variable may be configured incorrectly. Check if you have written the correct <INSTALL_DIR> or tried to activate it in the right folder.
    - If you added it to a `.bashrc` file, make sure that the command is correctly written and the file is found in the `~/.bashrc` folder.
 
-- For Python developers, if you installed OpenVINO using the installer previously, and now are installing OpenVINO with PIP, remove all the PATH settings and the lines with `setupvars` from `.bashrc`.
+- For Python developers, if you installed OpenVINO using the installer previously, and now are installing OpenVINO with PIP, remove all the PATH settings and the lines with `setupvars` from `.bashrc`. <!--updated-->
```
Comment (helena-intel) [inline docs/install_guides/troubleshooting-steps.md:55]: Maybe move this up and clarify that if you install OpenVINO with pip in a virtual environment, you do not need to set environment variables.

Comment (xu-yuan1) [reply docs/install_guides/troubleshooting-steps.md:55]: Done.

File: docs/install_guides/troubleshooting-steps.md [hunk 3]
```diff
@@ -8,33 +8,28 @@ If you run into issues while installing or configuring OpenVINO™, you can try
...
-* To check the version of OpenVINO Runtime, use the following command:
+* To check the version of OpenVINO Runtime, use the following code:
    ```sh
    from openvino.runtime import get_version get_version()
    ```
 
 ## Check the versions of Python and PIP
 
-To check your Python version, run `python -VV` or `python --version`. The supported Python versions should be 64-bit and between 3.7 and 3.9. Note that Python 3.6 is not supported anymore.
+To check your Python version, run `python -VV` or `python --version`. The supported Python versions should be 64-bit and between 3.7 and 3.9. Note that Python 3.6 is not supported anymore. <!--check with Ryan-->
```
Comment (ryanloney) [inline docs/install_guides/troubleshooting-steps.md:18]:
```suggestion
To check your Python version, run `python -VV` or `python --version`. The supported Python versions should be 64-bit and between 3.7 and 3.9. Starting in 2022.2, Python 3.6 is no longer recommended. Please upgrade.
```

File: docs/install_guides/troubleshooting-steps.md [hunk 4]
```diff
@@ -8,33 +8,28 @@ If you run into issues while installing or configuring OpenVINO™, you can try
...
 If your Python version does not meet the requirements, update Python:
 
 * For Windows, **do not install Python from a Windows Store** as it can cause issues. You are highly recommended to install Python from <https://www.python.org/>.
-* For Ubuntu and other Linux systems, use the Python version comes with the system, or install the `libpython3.X` libraries via the following commands (taking Python 3.7 as an example):
-```sh
-sudo apt-get install libpython3.7
-sudo apt-get install libpython3-dev
-```
-* For macOS, download a proper Python version from <https://www.python.org/> and install it. Note that macOS 10.x comes with python 2.7 installed, which is not supported, so you still need install Python from its official website.
+* For Linux and macOS systems, download and install a proper Python version from <https://www.python.org/>. See the [Python Beginners' Guide](https://wiki.python.org/moin/BeginnersGuide/Download) for more information on  version selection. Note that macOS 10.x comes with python 2.7 installed, which is not supported, so you still need install Python from its official website.
```
Comment (ryanloney) [inline docs/install_guides/troubleshooting-steps.md:23]:
```suggestion
* For Linux and macOS systems, download and install a proper Python version from <https://www.python.org/>. See the [Python Beginners' Guide](https://wiki.python.org/moin/BeginnersGuide/Download) for more information selecting a version. Note that macOS 10.x comes with python 2.7 installed, which is not supported, so you must install Python from the official website.
```

File: docs/install_guides/troubleshooting-steps.md [hunk 5]
```diff
@@ -45,31 +40,44 @@ For C++ developers with Windows systems, make sure that Microsoft Visual Studio
 
 ## Check if environment variables are set correctly 
 
-- If you have installed OpenVINO before, you probably have added `setupvars` to your `PATH /.bashrc` or Windows environment variables. After restarting your environment, you should see similar information as below:
+- For Python developers, if you installed OpenVINO using the installer previously, and now are installing OpenVINO with PIP, remove all the PATH settings and the lines with `setupvars` from `.bashrc`. Note that if you installed OpenVINO with PIP in a virtual environment, you do not need to set environment variables.
```
Comment (ryanloney) [inline docs/install_guides/troubleshooting-steps.md:43]:
```suggestion
- For Python developers, if you previously installed OpenVINO using the installer, and are now installing OpenVINO using PIP, please remove all the PATH settings and the lines with `setupvars` from `.bashrc`. Note that if you installed OpenVINO with PIP in a virtual environment, you don't need to set any environment variables.
```

File: docs/install_guides/troubleshooting-steps.md [hunk 6]
```diff
@@ -45,31 +40,44 @@ For C++ developers with Windows systems, make sure that Microsoft Visual Studio
...
+- If you have installed OpenVINO before, you probably have added `setupvars` to your `PATH /.bashrc` or Windows environment variables. After restarting your environment, you should see similar information as below: 
 ```sh
 [setupvars.sh] OpenVINO™ environment initialized
 ```
-   - If you don't see the information, your PATH variable may be configured incorrectly. Check if you have written the correct <INSTALL_DIR> or tried to activate it in the right folder.
+   - If you don't see the information, your PATH variables may be configured incorrectly. Check if you have written the correct <INSTALL_DIR> or tried to activate it in the right folder.
```
Comment (ryanloney) [inline docs/install_guides/troubleshooting-steps.md:48]:
```suggestion
   - If you don't see the information above, your PATH variables may be configured incorrectly. Check if you have typed the correct <INSTALL_DIR> or check if you are trying to activate in the correct directory.
```

File: docs/install_guides/troubleshooting-steps.md [hunk 7]
```diff
@@ -45,31 +40,44 @@ For C++ developers with Windows systems, make sure that Microsoft Visual Studio
...
    - If you added it to a `.bashrc` file, make sure that the command is correctly written and the file is found in the `~/.bashrc` folder.
 
-- For Python developers, if you installed OpenVINO using the installer previously, and now are installing OpenVINO with PIP, remove all the PATH settings and the lines with `setupvars` from `.bashrc`.
-
 ## Verify if OpenVINO is correctly installed
```
Comment (ryanloney) [inline docs/install_guides/troubleshooting-steps.md:51]:
```suggestion
## Verify that OpenVINO is correctly installed
```

File: docs/install_guides/troubleshooting-steps.md [hunk 8]
```diff
@@ -45,31 +40,44 @@ For C++ developers with Windows systems, make sure that Microsoft Visual Studio
...
-python -c "from openvino.runtime import Core"
-```
-If OpenVINO was successfully installed, nothing will happen. If not, an error will be displayed. 
+* For Python developers, to verify if OpenVINO is correctly installed, use the following command:
+   ```sh
+   python -c "from openvino.runtime import Core"
+   ```
+   If OpenVINO was successfully installed, nothing will happen. If not, an error will be displayed.
+
+* If you installed OpenVINO Runtime by using the installer, you can search "openvino" in Apps & Features on a Windows system, or check your installation directory on Linux to see if OpenVINO is there.
```
Comment (ryanloney) [inline docs/install_guides/troubleshooting-steps.md:59]:
```suggestion
* If you installed OpenVINO Runtime using the installer, you can search "openvino" in Apps & Features on a Windows system, or check your installation directory on Linux to see if OpenVINO is there.
```

File: docs/install_guides/troubleshooting-steps.md [hunk 9]
```diff
@@ -45,31 +40,44 @@ For C++ developers with Windows systems, make sure that Microsoft Visual Studio
...
+
+* If you installed OpenVINO Runtime from APT, use the `apt list --installed | grep openvino` command to list the installed OpenVINO packages.
+
+* If you installed OpenVINO Runtime from YUM, use the **yum list installed 'openvino\*'** command to list the installed OpenVINO packages.
 
 ## Check if GPU drvier is installed
 
 [Additional configurations](configurations-header.md) are required in order to use OpenVINO on different hardware.
```
Comment (ryanloney) [inline docs/install_guides/troubleshooting-steps.md:67]:
```suggestion
[Additional configurations](configurations-header.md) may be required in order to use OpenVINO with different hardware like Intel GPUs.
```

File: docs/install_guides/troubleshooting-steps.md [hunk 10]
```diff
@@ -45,31 +40,44 @@ For C++ developers with Windows systems, make sure that Microsoft Visual Studio
...
 
 To run inference on GPU, make sure that you have installed the correct GPU driver. To check that, see [additional configurations for GPU](configurations-for-intel-gpu.md).
```
Comment (ryanloney) [inline docs/install_guides/troubleshooting-steps.md:69]:
```suggestion
To run inference on an Intel GPU, make sure that you have installed the correct GPU driver. To check that, see [additional configurations for GPU](configurations-for-intel-gpu.md).
```

File: docs/install_guides/troubleshooting-steps.md [hunk 11]
```diff
@@ -45,31 +40,44 @@ For C++ developers with Windows systems, make sure that Microsoft Visual Studio
...
 
-## Firewall/Network Issues
+## Check firewall/network settings
```
Comment (ryanloney) [inline docs/install_guides/troubleshooting-steps.md:71]:
```suggestion
## Check firewall and network settings
```

**Review by ryanloney — CHANGES_REQUESTED** [review]

**Comment by ryanloney** [comment]
Troubleshooting issues seems very specific to VPU/HDDL and PRC, while troubleshooting-steps seems more general. Is this intentional? @xu-yuan1  @helena-intel

**Comment by xu-yuan1** [comment]
> Troubleshooting issues seems very specific to VPU/HDDL and PRC, while troubleshooting-steps seems more general. Is this intentional? @xu-yuan1 @helena-intel

Thanks for your comments Ryan! I updated some of the wordings to match the common style. Regarding the troubleshooting issues, the VPU/HDDL issues were inherited from previous versions, and the PRC issue was added since last release because I reported it. They are all we have now. :)

**Comment by xu-yuan1** [comment]
@ryanloney @helena-intel Hi both, I've updated with your comments. Could you help check and approve this PR? Then hopefully we can have the latest information in our nightly build next week. Thanks. :)

**Review by kblaszczak-intel — APPROVED** [review]
I believe it is fine. Further changes can be made in future PRs.

**Review by helena-intel — APPROVED** [review]

**Pull request merged**